howto/troubleshoot/dashboard-issues.md

```diff
@@ -16,32 +16,32 @@ A session does not start and the session details page displays the following err
 
 ![Session does not start|690x440](https://assets.ubuntu.com/v1/52b32f73-session-does-not-start.png)
 
-[Check the container logs](https://discourse.ubuntu.com/t/how-to-view-the-container-logs/24329) to find reasons for the session failure.
+[Check the instance logs](https://discourse.ubuntu.com/t/24329) to find reasons for the session failure.
 
 
 ## Instances(s) in Error status
 *Applies to: Anbox Cloud, Anbox Cloud Appliance*
 
 The **Instances** list page shows instances with Error status.
 
-![Container in Error status|690x440](https://assets.ubuntu.com/v1/aaf1194c-container-list-error.png)
+![Instance in Error status|690x440](https://assets.ubuntu.com/v1/aaf1194c-container-list-error.png)
 
 An instance can end up with an error status due to various reasons. It may not always be simple and easy to resolve this because of the variable factors involved, for example, the application that the instance is hosting or any installed addons.
 
 Click on the corresponding **INSTANCE ID** and check the instance details page for any possible error messages. The instance details page has an **Error Message** field that could be useful.
 
-![Container details page|690x440](https://assets.ubuntu.com/v1/0ff0d3ff-container-details-error.png)
+![Instance details page|690x440](https://assets.ubuntu.com/v1/0ff0d3ff-container-details-error.png)
 
 The **Error Message field** can give you a starting point for identifying the issue. Some reasons for an instance to go into error status could be:
 * Insufficient resources. Refer [Capacity planning](https://discourse.ubuntu.com/t/capacity-planning/28717).
-* Occasionally, access to Ubuntu archives could be a problem when creating an application. As an immediate workaround, you could disable the security update by running `amc config set container.security_updates false` or explicitly set `amc config set container.api_mirror <mirror_address>` to configure an instance to use a different APT mirror. See [AMS configuration](https://discourse.ubuntu.com/t/ams-configuration/20872) for more details.
+* Occasionally, access to Ubuntu archives could be a problem when creating an application. As an immediate workaround, you could disable the security update by running `amc config set instance.security_updates false` or explicitly set `amc config set instance.api_mirror <mirror_address>` to configure an instance to use a different APT mirror. See [AMS configuration](https://discourse.ubuntu.com/t/ams-configuration/20872) for more details.
```

Comment:
Is there a need to add a deprecated marker or something to say that the old still works, like it is done below for the API endpoints ?

Reply:
It should be marked as deprecated in the `ams-configuration.yaml` file I would say and then affects the generated table to mark it as deprecated.

Reply:
Updated the `ams-configuration.yaml` to add a field for deprecated